drivers/built_for_zero_report/app/views/built_for_zero_report/warehouse_reports/bfz/_filter.haml

```diff
@@ -9,6 +9,9 @@
         = f.input :section, as: :select_two, collection: BuiltForZeroReport.sections, selected: @section_key
 
   - content_for :filter_actions do
-    = f.submit 'Update Report', class: ['btn', 'btn-primary'], data: { disable_with: false }
+    .d-flex.w-100
+      = f.submit 'Update Report', class: ['btn', 'btn-primary'], data: { disable_with: false }
+      .ml-auto
+        = f.submit 'Submit Report', class: ['btn', 'btn-secondary'], data: { disable_with: false, confirm: 'Submitting this report will send data for all sections to Built For Zero via their submission API. Are you sure you would like to proceed?' }
```

Comment:
Should this be commented out until we are ready to release?

Reply:
Oh! Looks like I implemented the submit.  Should be good to leave this in.